internal/flowrate/flowrate.go

```diff
@@ -29,9 +29,9 @@ type Monitor struct {
 	sBytes int64         // Number of bytes transferred since sLast
 	sLast  time.Duration // Most recent sample time (stop time when inactive)
 	sRate  time.Duration // Sampling rate
-
-	tBytes int64         // Number of bytes expected in the current transfer
```

Comment:
I understand we're removing this because we don't use it

Reply:
I removed the `SetTransferSize()` method and all associated fields.

Reply:
I have undone this, as it does not behave to this PR.

Reply:
(y)